.vscode/settings.json

```diff
@@ -2,4 +2,106 @@
     "rust-analyzer.check.overrideCommand": [
         "cargo", "clippy", "--target-dir", "./target/clippy"
     ],
+    "files.associations": {
```

Comment:
These should probably go in the user settings.json instead of the workspace one due to how many there are: https://code.visualstudio.com/docs/getstarted/settings#_settings-file-locations

Reply:
good catch, I'll delete all of this